src/ocamlorg/lib/package.mli

```diff
@@ -71,6 +76,8 @@ module Module_map = Module_map
 
 type state
 
+val get_state : state -> Info.t OpamPackage.Version.Map.t OpamPackage.Name.Map.t
```

Comment:
Do we need to expose this in the interface?

Reply:
It doesn't look likes its useful for now so I've removed it